pkg/webui/console/views/overview/index.js

```diff
@@ -77,7 +77,7 @@ const componentMap = {
     return {
       applicationCount: selectApplicationsTotalCount(state),
       gatewayCount: selectGatewaysTotalCount(state),
-      fetching: createFetchingSelector([GET_APPS_LIST, GET_GTWS_LIST])(state),
+      fetching: createFetchingSelector([GET_APPS_LIST_BASE, GET_GTWS_LIST_BASE])(state),
```

Comment:
Please move it outside of `connect` so the function is not created any time the store gets updated.

```js
const overviewFetchingSelector = createFetchingSelector([GET_APPS_LIST_BASE, GET_GTWS_LIST_BASE])
```

and then simply use it